nirodents/workflows/brainextraction.py

```diff
@@ -53,121 +59,193 @@ def init_rodent_brain_extraction_wf(
         Run an extra step to refine the brain mask using a brain-tissue segmentation with Atropos.
 
     """
-
-    if omp_nthreads is None or omp_nthreads < 1:
-        omp_nthreads = cpu_count()
-
     inputnode = pe.Node(
         niu.IdentityInterface(fields=["in_files", "in_mask"]), name="inputnode"
     )
+    outputnode = pe.Node(
+        niu.IdentityInterface(fields=["out_corrected", "out_brain", "out_mask"]),
+        name="outputnode"
+    )
 
-    # Find images in templateFlow
+    # Find a suitable target template in TemplateFlow
     tpl_target_path = get_template(
         in_template,
-        resolution=debug + 1,
+        resolution=2,
```

Comment:
Seems pointless to use resolution 1. It's two orders of magnitude higher.

Reply:
I suppose this is dependent on the template chosen. Fine while we only have WHS, but other templates might not have ridiculously high starting resolutions.